test/ussd_chw.test.js

```diff
@@ -169,6 +169,59 @@ describe("app", function() {
         });
         // end re-dial flow tests
 
+        describe("dialback sms testing", function() {
+            it("send if redial sms not yet sent", function() {
```

Comment:
I'd like to see explicit testing for when an identity loads with and without the `chw.redial_sms_sent` field, and similar for other apps. You don't need to complete the whole flow, just this part.